src/AzureDevOpsPolicyConfigurator/Interfaces/ILogger.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace AzureDevOpsPolicyConfigurator
+{
+    /// <summary>
+    /// ILogger interface.
+    /// </summary>
+    internal interface ILogger
+    {
+        /// <summary>
+        /// Logs an ingormation message.
+        /// </summary>
+        /// <param name="message">message</param>
+        void Info(string message);
+
+        /// <summary>
+        /// Logs a debug message.
+        /// </summary>
+        /// <param name="message">message</param>
```

Comment:
The comment for `message` can be removed as `message` won't provide additional help in terms of describing the param.